### ISSUE TITLE
Upgrade to latest version of THREE.JS

### ISSUE THREAD

**Issue by madmaw** [nascherman/three-mtl-loader#2]
The plugin references an outdated version of THREE in the package.json file. This version mismatch causes the following error when run against version 0.87.1 of THREE

Uncaught TypeError: Cannot read property 'toString' of undefined at WebGLPrograms.getProgramCode 

If you update the package version, the problem disappears and textures load correctly

**Comment by ethanrose** [comment]
Also having the same issue / error message. Upgraded THREE to 0.88.0 and I still get the same errors

**Comment by ethanrose** [comment]
Actually looks like the assets being requested as XHRs rather than images

**Comment by wtlyu** [comment]
Yes. please update three version. 0.87.1 got another `Uncaught TypeError: Cannot read property 'elements' of undefined ` and 0.88 solves it.
